### PR TITLE
add permission test[wip]

### PR DESCRIPTION
**Pull request by ouwenkg**
### Changes

- refactor integrate py tests 
- add permission denied test

### Applicable Issues

#650

### CODE REVIEW

File: tests/integrate_test/cita_charge_mode.sh [hunk 1]
```diff
@@ -74,25 +81,29 @@ main() {
     test_fee_back 0
     echo "DONE"
 
-    echo -n "5) Run charge mode tests in v0 ...  "
+    echo -n "5) Check permission denied ... "
```
Comment (kaikai1024) [inline tests/integrate_test/cita_charge_mode.sh]: Why does the test be here?

File: tests/integrate_test/test_perm_denied.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+"""
+import functools
+import subprocess
+import time
+from jsonrpcclient.http_client import HTTPClient
+
+LATEST_VERSION = 1
+
+
+def send_tx(privkey, code="", version=LATEST_VERSION):
```
Comment (kaikai1024) [inline tests/integrate_test/test_perm_denied.py]: Would we refactor the `send_tx`, `get_receipt` and `rpc request` first?
Check the `tests/integrate_test/*.py`

Comment (ouwenkg) [reply tests/integrate_test/test_perm_denied.py]: Yeal, There should be a `utils.py`, contains these functions.

Reorganize the whole integrate test directory firstly.

File: tests/integrate_test/cita_charge_mode.sh (hunk 1 above)
Comment (ouwenkg) [reply tests/integrate_test/cita_charge_mode.sh]: And place better?

Comment (kaikai1024) [reply tests/integrate_test/cita_charge_mode.sh]: I'll check it later

Comment (ouwenkg) [reply tests/integrate_test/cita_charge_mode.sh]: I will refact these tests together, wait some time.

Comment (kaikai1024) [reply tests/integrate_test/cita_charge_mode.sh]: ok

**Pull request closed without merging**